addons/repair/models/repair.py

```diff
@@ -47,13 +47,13 @@ class Repair(models.Model):
     default_address_id = fields.Many2one('res.partner', compute='_compute_default_address_id')
     state = fields.Selection([
         ('draft', 'Quotation'),
-        ('cancel', 'Cancelled'),
         ('confirmed', 'Confirmed'),
         ('under_repair', 'Under Repair'),
         ('ready', 'Ready to Repair'),
         ('2binvoiced', 'To be Invoiced'),
         ('invoice_except', 'Invoice Exception'),
-        ('done', 'Repaired')], string='Status',
+        ('done', 'Repaired'),
+        ('cancel', 'Cancelled')], string='Status',
```

Comment:
useless diff here?

Reply:
Needed to display the cancel at last of all stages.

Reply:
oh okay, i haven't thought of that. thanks